tests/psa-client-server/psasim/src/aut_psa_random.c

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+ */
+
+#include "mbedtls/build_info.h"
+
+#include <psa/crypto.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "mbedtls/entropy.h"
+
+#define BUFFER_SIZE 100
+
+#if !defined(MBEDTLS_PSA_CRYPTO_CLIENT)
+int psa_random_main(void)
+{
+    printf("MBEDTLS_PSA_CRYPTO_C not defined.\r\n");
```

Comment:
I notice here the message doesn't match the `#if`, I haven't checked others... We probably also don't need `\r` - that's only needed on Windows if you compile with binary mode (rather than text mode), and we're not going to run this stuff on Windows

Reply:
Yeah, I was actually planning to remove all those checks: here we want to check _all_ PSA features through PSASIM, so they must all be available. I will update commits